test-fixture/integration-reactive/src/main/kotlin/com/linecorp/kotlinjdsl/test/reactive/querydsl/predicate/AbstractCriteriaQueryDslPredicateIntegrationTest.kt

```diff
@@ -302,4 +305,57 @@ abstract class AbstractCriteriaQueryDslPredicateIntegrationTest<S> : CriteriaQue
         // then
         assertThat(orderItemIds).isEmpty()
     }
+
+    @Test
+    fun `exists - subquery`() = runBlocking {
+        val existFoundOrders = withFactory { queryFactory ->
+            queryFactory.listQuery<Order> {
```

Comment:
@heli-os I am really appreciate your feedback! but I didn't understand this part...ㅜㅜ can you give me more detail?

Reply:
@heli-os Yes maybe now a lot of people already used `subquery` function. so It would be hard to change convention. how about adding subQuery function and mark `@Deprecated` annotation on `subquery` function, if project owner want to fix that one

CC @cj848

Reply:
'subquery' is the same as the name on criteria API, and in fact, if you look at other codes than subQuery, it is often used as a subquery.